last_seen.py

```diff
@@ -11,23 +11,41 @@ def seen_evt(self, event):
         events = ['quit', 'part', 'kick', 'join', 'rename', 'topic', 'message']
         if event.event_type in events:
             nick = event.meta['nick']
-            self.seen[nick] = (event.event_type, gmtime())
+            self.seen[nick] = (event.event_type, datetime.utcnow())
 
     @cmd
     def last_seen(self, event):
-        """
-        Returns when was given user last seen.
+        """Return last activity and its time of given user.
 
-        Example: !last_seen <user>
+        Examples:
+            !last_seen <user>
         """
         args = event.args
         if len(args) < 1:
-            return "Name argument missing."
+            return 'Nick not specified. (usage: {0}last_seen <nick>)'.format(
+                    event.source_bot.command_token)
```

Comment:
@Adman this thing seems to cause the travis build to fail but we do know why and this is also just a temporary solution so it is safe to ignore it.